carbon/next.config.js

```diff
@@ -12,22 +12,6 @@ module.exports = async (phase, { defaultConfig }) => {
     reactStrictMode: true,
     //Provide source maps on preview deployments
     productionBrowserSourceMaps: !IS_PRODUCTION,
-    async rewrites() {
-      return [
-        {
-          source: "/:locale",
-          destination: "/:locale/overview",
-        },
-        {
-          source: "/:locale/token-details",
-          destination: "/:locale/token-details",
-        },
-        {
-          source: "/:locale/retirement_trends",
-          destination: "/:locale/retirement_trends/by-pool",
-        },
-      ];
-    },
```

Comment:
What does this change achieve?

Reply:
It removes the rewrites. There is an optimization in nextJS that replaces internal links so the page is not reloaded and the site works as a single page app. But I found out that this breaks with rewrites. That's why the Overview link was not working sometimes.
So the first one is replaced by the file "app/[locale]/page.tsx"
And the other two were no longer used.